src/components/skills-quiz/tests/SkillsQuizStepper.test.jsx

```diff
@@ -304,15 +272,45 @@ describe('<SkillsQuizStepper />', () => {
       </AppContext.Provider>,
       { route: '/test/skills-quiz/?skill_names=xyz' },
     );
+    userEvent.click(screen.getByText(GOAL_DROPDOWN_DEFAULT_OPTION));
+    userEvent.click(await screen.findByText(DROPDOWN_OPTION_GET_PROMOTED));
+    expect(screen.getByText(SKILLS_FACET.title)).toBeInTheDocument();
+    expect(screen.getByText(CURRENT_JOB_FACET.title)).toBeInTheDocument();
+    expect(screen.getByText(DESIRED_JOB_FACET.title)).toBeInTheDocument();
+  });
+
+  it('Handles removal skill is handled correctly.', async () => {
+    const searchContext = {
+      refinements: { skill_names: ['test-skill-1', 'test-skill-2'] },
+      dispatch: () => null,
+    };
+
+    renderWithRouter(
+      <AppContext.Provider value={defaultAppState}>
+        <UserSubsidyContext.Provider value={defaultUserSubsidyState}>
+          <SubsidyRequestsContext.Provider value={defaultSubsidyRequestState}>
+            <SearchContext.Provider value={{ ...searchContext }}>
```

Comment:
Example of a needed 'Provider of Providers'.  Or maybe a test utility in the future.

Reply:
Yeah, definitely defer-worthy for now, though. Just updating these existing tests as minimally as possible for these changes haha.